src/index.html

```diff
@@ -4,9 +4,348 @@
+              class="nav__link"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__ltem">
+            <a href="#Gadgets"
+              class="nav__link">Gadgets</a>
+          </li>
+
+          <li class="nav__ltem">
+            <a href="#Tablets"
+              class="nav__link">Tablets</a>
+          </li>
+
+          <li class="nav__ltem">
+            <a href="#Photo"
+              class="nav__link">Photo</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#Video"
+              class="nav__link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <div class="cards">
+        <div data-qa="card" class="card cards__card">
+          <img class="card__image"
+            src="/src/images/imac.jpeg"
+            alt="iMac A1419">
+
+          <h2 class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <p class="card__code">Product code: 195434</p>
+
+          <div class="card__review-container">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+
+            <p class="card__review-count">Reviews: 5</p>
+          </div>
+
+          <div class="card__price-container">
+            <h3 class="card__price-label">Price:</h3>
+            <p class="card__price">$2,199</p>
+          </div>
+
+          <a class="card__buy"
+            href="#"
+            data-qa="card-hover"
+          >
+            Buy
+          </a>
+        </div>
+
+        <div class="card cards__card">
+          <img class="card__image"
+            src="./images/imac.jpeg"
+            alt="iMac A1419">
+
+          <h2 class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <div class="card__code">Product code: 195434</div>
+
+          <div class="card__review-container">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+
+            <div class="card__review-count">Reviews: 5</div>
+          </div>
+
+          <div class="card__price-container">
+            <h3 class="card__price-label">Price:</h3>
+            <p class="card__price">$2,199</p>
+          </div>
+
+          <a class="card__buy"
+            href="#"
+          >
+            Buy
+          </a>
+        </div>
+
+        <div class="card cards__card">
+          <img class="card__image"
+            src="./images/imac.jpeg"
+            alt="iMac A1419">
```

Comment:
<img width="357" alt="attrCheklist" src="https://github.com/mate-academy/layout_catalog/assets/77551820/39192238-368f-4b57-b2fc-a0178bcb7bb3">

**PLS DO THE SAME FOR ALL SUCH CASES IN THE PROJECT**

Reply:
should I do this only for those elements that have 3 or more attributes?